hadoop-tools/hadoop-aliyun/src/main/java/org/apache/hadoop/fs/aliyun/oss/Constants.java

```diff
@@ -211,4 +211,8 @@ private Constants() {
   public static final String LIST_VERSION = "fs.oss.list.version";
 
   public static final int DEFAULT_LIST_VERSION = 2;
+
+  // oss redirect
+  public static final String REDIRECT_ENABLE_KEY = "fs.oss.redirect.enable";
```

Comment:
add javadocs with `{@value}` to show the value...really helps in IDE use

Reply:
> add javadocs with `{@value}` to show the value...really helps in IDE use

OK。I will add javadocs in the next commit。